doc/source/serve/monitoring.md

```diff
@@ -82,18 +111,21 @@ class Silenced:
         logger.setLevel(logging.ERROR)
 ```
 
-This will prevent the replica INFO-level logs from being written to STDOUT or to files on disk.
+This controls which logs are written to STDOUT or files on disk.
 You can also use your own custom logger, in which case you'll need to configure the behavior to write to STDOUT/STDERR, files on disk, or both.
 
```

Comment:
Apart from standard Python logger, Serve supports custom logging. Custom logging via configuration can control the behavior of what messages are written to STDOUT/STDERR, files on disk, or both.